parsers/verilog.c

```diff
@@ -122,7 +124,8 @@ static kindDefinition SystemVerilogKinds [] = {
  { true, 'P', "program",   "programs" },
  { false,'Q', "prototype", "prototypes" },
  { true, 'R', "property",  "properties" },
- { true, 'T', "typedef",   "type declarations" }
+ { true, 'T', "typedef",   "type declarations" },
+ { true, 'v', "member",    "member elements" },
```

Comment:
Designing kinds is the most important task in ctags development.
I don't know Verilog. So if you are o.k., I'm o.k.
@RadekRR, this is the chance to reflect your idea to the software you are using:-).